Extensions/dnSpy.Analyzer/TreeNodes/EventOverriddenNode.cs

```diff
@@ -0,0 +1,51 @@
+﻿using System;
+using System.Collections.Generic;
+using System.Threading;
+
+using dnlib.DotNet;
+
+using dnSpy.Analyzer.Properties;
+using dnSpy.Contracts.Decompiler;
+using dnSpy.Contracts.Text;
+
+namespace dnSpy.Analyzer.TreeNodes {
+	sealed class EventOverriddenNode : SearchNode {
+		readonly EventDef analyzedEvent;
+
+		public EventOverriddenNode(EventDef analyzedEvent) {
+			if (analyzedEvent == null)
+				throw new ArgumentNullException(nameof(analyzedEvent));
+
+			this.analyzedEvent = analyzedEvent;
+		}
+
+		protected override void Write(ITextColorWriter output, IDecompiler decompiler) =>
+			output.Write(BoxedTextColor.Text, dnSpy_Analyzer_Resources.OverridesTreeNode);
+
+		protected override IEnumerable<AnalyzerTreeNodeData> FetchChildren(CancellationToken ct) {
```

Comment:
It doesn't find all events it overrides since it only checks the base type instead of all base types.